tests/test_tasks/test_all_abstasks.py

```diff
@@ -66,6 +67,9 @@ def test_load_data(
             mock_dataset_transform.assert_called_once()
 
 
+@pytest.mark.skipif(
+    condition=os.getenv("NO_SKIP", 0) == 1, reason="Just in separate CI Job"
+)
```

Comment:
Hmm, I would probably use:


```suggestion
@pytest.mark.test_datasets
```

which is then run using:
```
pytest -m pytest.mark.test_datasets
```

[docs](https://docs.pytest.org/en/stable/example/markers.html)